bn_mp_dr_reduce.c

```diff
@@ -58,7 +58,7 @@ int mp_dr_reduce(mp_int *x, const mp_int *n, mp_digit k)
    *tmpx1++ = mu;
 
    /* zero words above m */
-   MP_ZERO_DIGITS(tmpx1, x->used - m - 1);
+   MP_ZERO_DIGITS(tmpx1, (x->used - m) - 1);
```

Comment:
I mostly agree with this explicit parens, but these are weird. Are you also adding parens to `(a + b) + c`?

Reply:
here the full rational of my linter:
```
       Operator 'Name' followed by operator 'Name' is confusing.  Use
       parentheses. -- Some combinations of operators seem to be
       confusing.  For example

              a = b - c - d;
              a = b - c + d;
              a = b / c / d;
              a = b / c * d;

       tend to befuddle the reader.  To reduce confusion we recommend
       using parentheses to make the association of these operators
       explicit.  For example:

              a = (b - c) - d;
              a = (b - c) + d;
              a = (b / c) / d;
              a = (b / c) * d;

       in place of the above.
```

Reply:
Ok thx! It seems I am more befuddled by the parens. But I am well aware of the relative precedences. If I want parens I use lisp ;)